docs/configuration/configuration.md

```diff
@@ -1464,6 +1468,72 @@ See below for the configuration options for Lightsail discovery:
 [ port: <int> | default = 80 ]
 ```
 
+### `<linode_sd_config>`
+
+Linode SD configurations allow retrieving scrape targets from [Linode's](https://www.linode.com/)
+Linode APIv4.
+This service discovery uses the public IPv4 address by default, by that can be
+changed with relabelling, as demonstrated in [the Prometheus linode-sd
+configuration file](/documentation/examples/prometheus-linode.yml).
```

Comment:
Maybe we should document the scopes required to make the token in the Credentials section.